source/_integrations/verisure.markdown

```diff
@@ -87,3 +87,19 @@ automation:
 | disable_autolock | Disables autolock function for a specific lock. |
 | enable_autolock | Enables autolock function for a specific lock. |
 | smartcam_capture | Capture a new image from a specific smartcam. |
+
+## Lock
+| State | Description |
+| ------- | ----------- |
+| locked | Lock is locked |
+| unlocked | Lock is unlocked |
```

Comment:
These describe states of the Lock entity, which should be handled by the Lock documentation, not the integration-specific documentation.

Reply:
I see, good point. Ok now?